src/CoolTime.cpp

```diff
@@ -857,3 +813,103 @@ String CoolTime::formatDigits(int digits) {
 
   return (String(digits));
 }
+
+/**
+ *  CoolTime::timePooConfigl()
+ *
+ *  utility method for chosing the server with the best ping
+ *  returns 0 if it fails or returns the number of the const char* timeServer[]
+ *
+ *  \return formatted string of the input digit
+ */
+int CoolTime::timePoolConfig() {
+
+#if DEBUG == 1
+  Serial.println();
+  Serial.println(F("Enter timePoolConfig"));
+  Serial.println();
+#endif
+
+  Serial.println(F("Please wait, performing NTP server test..."));
+  unsigned long timer[6];
+  bool timeout[6];
```

Comment:
even a (signed) int can do the job since we should not hit 32 000 ms with 5 calls and a timeout at 1000ms.. 
millis() is a UL, that's why.. needs less memory and is faster then casting all the stuff...

Reply:
OK but you are using a `uint32_t` for `beginWait` below, so this is not consistent. 

Plus:
* C-style casting does not take time or use memory as it's compile time only... So I don't see your point.
* `int` size is ambiguous : on AVR it's 16 bit, but here on ESP it's 32 bit (so max value of `signed int` is around 2 billions, not 32000 as you said).  Using newer `int8_t`, `int16_t`, `int32_t`, `int64_t` types is way more explicit

But again, my point is only to stay consistent use the same types for comparable values.